client/landing/site.landing/coffee/teams/findteamview.coffee

```diff
@@ -55,21 +57,32 @@ module.exports = class FindTeamView extends kd.TabPaneView
     utils.findTeam email,
       error       : (xhr) =>
         { responseText } = xhr
-        new kd.NotificationView { title : responseText }
+        @showNotification @formatServerError responseText
         @form.button.hideLoader()
       success     : =>
         @form.button.hideLoader()
         @form.reset()
 
-        new kd.NotificationView
-          cssClass : 'recoverConfirmation'
-          title    : 'Check your email'
-          content  : 'We\'ve sent you a list of your teams.'
-          duration : 4500
+        @showNotification 'Check your email', 'We\'ve sent you a list of your teams.'
 
         kd.singletons.router.handleRoute '/'
 
 
+  formatServerError: (err) ->
+
+    return 'You don\'t participate in any team'  if err is EMPTY_TEAM_LIST_ERROR
```

Comment:
The screenshot you mentioned is not available but I guess you meant this - http://take.ms/fuFaP. We can also change `You can sign in or join to the following teams:` to `You can sign in or join to the following team:` if user has the only team. Does it make sense?